pkgs/by-name/ru/rustlings/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  cargo,
+  rustc,
+  clippy,
+  makeWrapper,
+}:
+let
+  pname = "rustlings";
+  version = "6.2.0";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "rust-lang";
+    repo = "rustlings";
+    rev = "v${version}";
+    hash = "sha256-BVmiMIIx8YEO57FO0ZpsCQcim68mn7NHpAOO86dZqlI=";
+  };
+
+  cargoHash = "sha256-ZupwPw/bfeN+s7IWXbY21K/ODXSaYef+IDDpBVCi3Ek=";
+
+  # Disabled test that does not work well in an isolated environment
+  checkFlags = [
+    "--skip=run_compilation_success"
+    "--skip=run_test_success"
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    cargo
+    rustc
+    clippy
```

Comment:
Don't these need to be run on the buildPlatform and aren't these in buildRustPackage by default?

Reply:
Yeah they do - but looks like rustlings also requires that you have cargo, rust, and clippy in your path when you run it. Is there a way to get the same versions of cargo, rust, and clippy out of `buildRustPackage` and then place them in `wrapProgram` (maybe with `--suffix-each`)?